aws/eks/eks/run.go

```diff
@@ -76,10 +76,10 @@ func Run(ctx *pulumi.Context) error {
 		EndpointPublicAccess:         pulumi.BoolPtr(false),
 		Fargate:                      fargateProfile,
 		ClusterSecurityGroup:         clusterSG,
-		NodeAssociatePublicIpAddress: pulumi.BoolPtr(false),
+		NodeAssociatePublicIpAddress: pulumi.BoolRef(false),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
What is the difference between `BoolPtr` and `BoolRef` ? (Yes I could google the doc)

Reply:
They changed the type in EKS provider 1.0.0, it was `pulumi.Bool` (`BoolPtr`), now it's just `*bool` (`BoolRef`)